docs/conf.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
```

Comment:
```suggestion
import os
import sys
sys.path.insert(0, os.path.abspath('..'))
```

If you are going to import `sphinx_search` below, you should probably add the package to the path. While this should work on Read the Docs itself due to your `.readthedocs.yml`, this will make it easier for local builds.